core/src/main/java/org/elasticsearch/common/geo/GeoUtils.java

```diff
@@ -56,6 +65,30 @@
     /** Earth ellipsoid polar distance in meters */
     public static final double EARTH_POLAR_DISTANCE = Math.PI * EARTH_SEMI_MINOR_AXIS;
 
+    /** Returns true if value is neither NaN nor infinite. */
+    public static boolean isValidNumber(double value) {
```

Comment:
generic question, this doesn't seem like some geo specific util, wondering if it belongs here.